core/settings.talon

```diff
@@ -0,0 +1,54 @@
+-
+settings():
```

Comment:
Should we keep this at the top-level instead? It's fairly expected users will modify settings, and burying this in `core/` might mean a lot of people won't know about these.

Reply:
Yeah, I think putting settings.talon at top-level makes sense

Reply:
It might make sense to put it in a setting directory, so users can put all their setting—eg the cursorless settings—in one place.

Reply:
We do have `/settings` to hold CSVs but it is gitignored.

Reply:
Maybe we can autogenerate `settings.talon` in `/settings` if it doesn’t exist on launch?